ecl/hql/hqlfold.cpp

```diff
@@ -4285,6 +4285,10 @@ IHqlExpression * foldConstantOperator(IHqlExpression * expr, unsigned foldOption
                 return LINK(arg);
             break;
         }
+    case  no_simplified:  // Only used during syntax checking:
```

Comment:
I don't think this code is needed any more now that simplified expression returns the expression for constants